### ISSUE TITLE
Definition of "Claimed Identity"

### ISSUE THREAD

**Issue by tarbrix** [usnistgov/800-63-3#130]
2:

Technology Provider:

63-3:

Section 3, Page 25, Claimed Identity :

First, the definition presumes the reader understands that the "claim" is within the context of identity proofing and not an identity assertion during authentication.  Secondly, when used for identity proofing, the combination of Name, DOB and _Current_ Address are typically 90-some percent reliable in uniquely resolving an identity within commercial databases.  The addition of other attributes such as SSN, previous address, gender (gasp), and whether or not "Name" is defined as Full First, Last and  Middle or simply First Initial + Last Name, etc.  can improve accuracy to the very high 90s.  With many commercial databases the terms Name, DOB and Address are slightly ambiguous.  They need to be more precisely defined to be useful.  For example, is Middle initial allowable in "Name"?  Does "address" include street, city, state and zip?   

Suggested Change:
1.  Either:  Fully specify the definitions: of Name (First, Last, Middle, - initials, prefix, suffix), DOB (MMDDYY, MMYY, etc.), and Address (full, street, etc.) _AND/OR_ 
2.  Do not precisely define "Claimed Identity" as "Name, DOB and Address" rather, define it more generically as a "claim of ownership of a set of identity attributes".  The identity proofer, consistent with the requirements of the particular Identity Assurance Level, will verify the claimed attributes can be resolved to a unique identity with XX% accuracy. 

---

Organization: 1 = Federal, 2 = Industry, 3 = Other

**Comment by paul-grassi** [comment]
@tarbrix please see current version at https://pages.nist.gov/800-63-3 as well as the updated 800-63A volume.  We removed the requirement that specifies the actual attributes that must be required.

**Comment by rsfaron23** [comment]
where is the latest red-lined version from the May 6 2016 version to be found? The individual github elements are hard to follow when trying to compare to the oldest version?

Robert Faron
202-905-7000
rsfaron@gmail.com

**Comment by paul-grassi** [comment]
Bob,
Unfortunately using this tool, we don't manage redline, word versions.  You can see the source files in GitHub, and look at the history of every one.  So in the case of 800-63-3 definitions, you can look [here](https://github.com/usnistgov/800-63-3/commits/nist-pages/sp800-63-3/sec3_definitions.md).

**Comment by rsfaron23** [comment]
Thanks I will figure it out. 

Bob Faron
202-905-7000
